modules/configuring-the-global-header/proc-mount-points.adoc

```diff
@@ -0,0 +1,59 @@
+[id="mount-points-for-dynamic-plugin-intergration_{context}"]
+= Mount points for dynamic plugin integration
+
+You can customize the application header in {product-short} using mount points for dynamic plugins. These mount points give flexibility in configuring the position of the header, its components and dropdown menus. You can create a customized experience with the following enhancements:
+
+application/header::
+Controls the header position. Use `config.position` to set placement as either `above-main-content` or `above-sidebar`.
+
+global.header/component::
+Configures header components. Use `config.priority` to set the order of components, and pass properties (including CSS styles) via `config.props`.
+
+.Adding a `Create` button
```

Comment:
Shall we rename `Create` to `Self-service` as there's a ticket [RHIDP-6164](https://issues.redhat.com/browse/RHIDP-6164) to do so?

```suggestion
.Adding a `Self-service` button
```